bin/picca_deltas.py

```diff
@@ -295,6 +301,21 @@ def cont_fit(data):
                         nb_dla_in_forest += 1
         log.write("Found {} DLAs in forests\n".format(nb_dla_in_forest))
 
+    ## Mask BALs
+    if not args.bal is None:
```

Comment:
the keyword is not called bal anymore. This is why the travis automatic tests are failing. 

Could you please make sure the modified code works well with and without these new options ?